GW2EIEvtcParser/EIData/ProfHelpers/Engineer/HolosmithHelper.cs

```diff
@@ -45,6 +45,11 @@ public static bool IsPhotonForgeTransform(long id)
             new Buff("Afterburner", Afterburner, Source.Holosmith, BuffStackType.StackingConditionalLoss, 25, BuffClassification.Other, BuffImages.SolarFocusingLens),
             new Buff("Heat Therapy", HeatTherapy, Source.Holosmith, BuffStackType.Stacking, 10, BuffClassification.Other, BuffImages.HeatTherapy),
             new Buff("Overheat", OverheatEffect, Source.Holosmith, BuffClassification.Other, BuffImages.Overheat),
+
+            // heat buffs only present in forge
+            new Buff("0-50 Heat", PhotonForgeAbove0Heat, Source.Holosmith, BuffClassification.Unknown, BuffImages.Heat),
```

Comment:
Why Unknown classification?

Reply:
Added icons & changed the names to e.g. "0-50 Heat (Photon Forge)". You think that's clear enough to keep displaying the percentage in the buffs tab?

Reply:
Looks good to me